src/components/SignInSteps/SignInStepsStepper.tsx

```diff
@@ -12,6 +12,15 @@ import { Multistepper } from '../Dialogs'
 export const SignInStepsStepper: React.FC = () => {
   const navigate = useNavigate()
   const step = Number(useRouterQuery('step'))
+  const [isStepperOpen, setIsStepperOpen] = useState(false)
+
+  useEffect(() => {
+    if (step >= 1 || step > 3) {
+      setIsStepperOpen(true)
+    } else {
+      setIsStepperOpen(false)
+    }
+  }, [step])
```

Comment:
What's the purpose of that?

Reply:
Without this `useEffect` if you just type in a browser `/studio/signin?step=1` the scroll will be not blocked, because `onEnter` will be not called.

Reply:
Oh, actually this won't be needed, I'll remove that